app/src/main/res/layout-sw600dp-port/completed_story_list_fragment.xml

```diff
@@ -30,7 +30,6 @@
         android:background="?attr/colorPrimary"
         android:fontFamily="sans-serif"
         android:minHeight="?attr/actionBarSize"
-        android:textSize="20sp"
```

Comment:
This toolbar is missing ` app:titleTextAppearance="@style/ToolbarTextAppearance"`

Reply:
done.

Reply:
Have you committed these changes yet?

Reply:
@ARJUPTA AND @Luffy18346 No the changes are not yet committed.

Reply:
@rt4914 and @ARJUPTA sorry it was my mistake. I had committed these changes in another branch. Now, I have committed in this branch.

Reply:
Thanks.